packages/api/endpoints/providers.js

```diff
@@ -159,34 +157,17 @@ async function put(req, res) {
     );
   }
 
-  try {
-    oldProvider = await providerModel.get({ id });
-  } catch (error) {
-    if (error.name !== 'RecordDoesNotExist') {
-      throw error;
-    }
-    log.warn(`Dynamo record for Provider ${id} not found, proceeding to update with PostgreSQL record alone`);
-  }
-
   apiProvider.updatedAt = Date.now();
-  apiProvider.createdAt = oldPgProvider.created_at.getTime();
+  apiProvider.createdAt = existingPgProvider.created_at.getTime();
 
   let record;
   const postgresProvider = await translateApiProviderToPostgresProvider(apiProvider);
 
-  try {
-    await createRejectableTransaction(knex, async (trx) => {
-      await providerPgModel.upsert(trx, postgresProvider);
-      record = await providerModel.create(apiProvider);
-      await indexProvider(esClient, record, process.env.ES_INDEX);
-    });
-  } catch (innerError) {
-    // Revert Dynamo record update if any write fails
-    if (oldProvider) {
-      await providerModel.create(oldProvider);
-    }
-    throw innerError;
-  }
+  await createRejectableTransaction(knex, async (trx) => {
+    const updatedPostgresProvider = await providerPgModel.upsert(trx, postgresProvider);
```

Comment:
```suggestion
    const [updatedPostgresProvider] = await providerPgModel.upsert(trx, postgresProvider);
```
example:
https://github.com/nasa/cumulus/blob/feature/rds-phase-3/packages/api/lib/writeRecords/write-execution.js#L137

Reply:
That's what I get for not running my tests. 🤦